packages/wallets/src/components/Dropzone/Dropzone.tsx

```diff
@@ -144,7 +143,7 @@ const Dropzone: React.FC<TProps> = ({
                                     <IconButton
                                         className='wallets-dropzone__remove-file'
                                         data-testid='dt_remove-button'
-                                        icon={<CloseIcon width={12} />}
+                                        icon={<LegacyClose2pxIcon iconSize='xs' width={12} />}
                                         onClick={removeFile}
                                         size='sm'
                                     />
```

Comment:
@george-usynin-deriv could you assist with removing the TODO here since the icons are updated 🙏